lib/repository.ts

```diff
@@ -67,6 +74,14 @@ export class RepositorySetup extends Construct {
           enforceAdmins: true,
           allowsDeletions: false,
           allowsForcePushes: false,
+          requiredPullRequestReviews: [
+            {
+              requiredApprovingReviewCount: 1,
```

Comment:
Isn't this problematic if we want pre-built provider updates to auto-merge or are they auto-approved?

Reply:
Mergify currently auto-approves them, and if we switch away from Mergify and use the other workflows, adding the "auto-approve" label to a PR will cause a bot to auto-approve the PR.